website/templates/add_record.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'base.html' %}
+{% block content %}
+
+<div class="col-md-6 offset-md-3">
+<h1>Add Record</h1>
+<br/>
+
+<form method="POST" action="{% url 'add_record' %}">
+	{% csrf_token %}
+
+	{{ form.as_p }}
+
+<br/>
+  <button type="submit" class="btn btn-secondary">Add Record</button>
+   <a href="{% url 'home' %}" class="btn btn-secondary">Back</a>
```

Comment:
I guess this is back button. so better use back url instead of home. This will it more dynamic and re-usable.